lms/djangoapps/discussion_api/views.py

```diff
@@ -158,7 +159,9 @@ class ThreadViewSet(DeveloperErrorViewMixin, ViewSet):
             "unanswered" for question threads with no marked answer. Only one
             can be selected.
 
-        The topic_id, text_search, and following parameters are mutually
+        * inline_discussion_id: ID of an inline discussion for any module
```

Comment:
@nasthagiri Is this the ID for a content-specific discussion topic that is added via a discussion component? (vs. a course-wide discussion topic)